### PR TITLE
add multicluster service mesh solution.

### PR DESCRIPTION
**Pull request by morvencao**
ref: https://github.com/open-cluster-management-io/OCM/issues/34

Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

**Comment by mikeshng** [comment]
Hi @yue9944882 , since you are familiar with this area. Could you please take a look? Thanks.

/assign yue9944882 

Hi @nyechiel , could you please take a quick look if you are interested? Thanks.

/assign nyechiel 

/assign qiujian16

File: solutions/run-multicluster-servicemesh/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+## Prerequisite
+
+Set up the dev environment in your local machine following [setup dev environment](../setup-dev-environment).
+
+## Install Multicluster Service Mesh Addon on OCM
+
+1. Clone the multicluster-mesh github repository and install:
+
+```bash
+git clone git@github.com:open-cluster-management-io/multicluster-mesh.git && cd multicluster-mesh
```
Comment (qiujian16) [inline solutions/run-multicluster-servicemesh/README.md:14]: I think we should build a helm chart and push to ocm helm repo, so we do not need to clone the code.

File: solutions/run-multicluster-servicemesh/README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+kubectl config use-context kind-hub
+kubectl apply -f ./manifests/meshfederation.yaml
+```
+
+## Verify Mesh Federtion with Bookinfo Application
+
+1. Deploy part(productpage,details,reviews-v1,reviews-v2,ratings) of the bookinfo application in cluster1:
+
+```bash
+kubectl config use-context kind-cluster1
```
Comment (qiujian16) [inline solutions/run-multicluster-servicemesh/README.md:68]: could we do it using clusteradm create work -f?

File: solutions/run-multicluster-servicemesh/README.md (hunk 1 above)
Comment (morvencao) [reply solutions/run-multicluster-servicemesh/README.md:14]: filed a new PR for this: https://github.com/open-cluster-management-io/helm-charts/pull/7

Comment (qiujian16) [reply solutions/run-multicluster-servicemesh/README.md:14]: I think this can be done by adding a github action in mesh repo?

Comment (morvencao) [reply solutions/run-multicluster-servicemesh/README.md:14]: thanks @qiujian16 
I updated the [PR](https://github.com/open-cluster-management-io/multicluster-mesh/pull/5/commits/e9af0c8fefa3f24abc2ec7fe4557c03c97c354cd) for the mesh repo to add release workflow, that workflow will upload teh chart tarball to the `helm-charts` repo.

File: solutions/run-multicluster-servicemesh/README.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# Set up a Multicluster Service Mesh on OCM
+
+This scripts is to setup an multicluster service mesh on top of OCM. The guide will bootstrap 3 kind clusters(hub, cluster1, and cluster2) on your local machine and then deploy the [multicluster mesh addon](https://github.com/open-cluster-management-io/multicluster-mesh). After that, creating service meshes from the hub to the managed clusters and finally federating the service mshes so that microservices deployed into different managed clusters can be access each other.
```
Comment (clyang82) [inline solutions/run-multicluster-servicemesh/README.md:3]: s/mshes/meshes/

Comment (morvencao) [reply solutions/run-multicluster-servicemesh/README.md]: nice catch! fixed.

File: solutions/run-multicluster-servicemesh/README.md (hunk 1 above)
Comment (morvencao) [reply solutions/run-multicluster-servicemesh/README.md]: Updated installation with helm chart from `https://openclustermanagement.blob.core.windows.net/releases/`

**Pull request merged**